### DISCUSSION TITLE
2021-06-02 Weekly Meeting 6 Agenda / Discussion

### DISCUSSION THREAD

**Discussion by robertlong** [omigroup/omigroup#41] [Weekly Meetings]
Weekly Meeting 6 is happening on Wednesday 6/2/2021 from 9:00 AM PDT - 10:00 AM PDT

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR Discord](https://discord.gg/NJtT9grz5E).

## Agenda
 - [W3C Community Group Chair Election](https://github.com/omigroup/OMI/discussions/17)
 - OMI co-development
 - [OMI Identity](https://github.com/omigroup/OMI/issues/36)
 - make sure to comment below with links to your proposals so we can get time scheduled to talk about them

**Comment by LightLodges** [comment]
Putting forward a few org development tools/resources that we may want to consider:

https://opencollective.com/  (open transparent accounting, free solution for orgs)

https://platform.coop/  (is this the type of decision making style we are interested in adopting for charter?)

Network Weaving (strategy and map analysis of our current network & trajectories)
https://networkweaver.com/store/  for free resources and see attached drive doc 
https://docs.google.com/presentation/d/1W-82gd5woXKMpA6B83MWmfrX9US0chzJ5KCZRXBLpns/edit#slide=id.g7aac9b8b44_0_0 for how and when/why to use this strategy for co-development structure and planning

**Comment by technobaboo** [comment]
https://github.com/omigroup/OMI/discussions/23

I want to bring this into the public consciousness so that we can get a useful spec out there to establish our process and fail faster.

**Comment by mrmetaverse** [comment]
_Some_ [notes](https://hackmd.io/@mrmetaverse/omi-060221) from todays meeting.
